### PR TITLE
EDU-1246 Reimplement document control panels

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1246

**The new badges are as follows** (and seem to fit better when accredited authors and protected are combined)
Screen Shot 2023-03-22 at 18 29 33

### CODE REVIEW

**Comment by ahelmberger** [comment]
When I check out this branch, it re-introduces code from the main branch at tag `v0.110.4` (which were later reverted on `main`). These changes do not appear in the "Files changed" on this PR, though :grimacing:
Could you please rebase or merge the latest `main` into this branch.
(You can check if the warning logging in `registered-plugin.js` disappears)

File: src/components/pages/document.yml [hunk 1]
```diff
@@ -38,9 +41,18 @@ document:
...
     de: Komentare
+  commentsActionTooltip:
+    en: Open comments mode
+    de: Komentar-Modus öffnen
   history:
     en: Version history
     de: Versionshistorie
+  historyActionTooltip:
+    en: Open version history mode
+    de: Versionshistorie-Modus öffnen
```
Comment (ahelmberger) [inline src/components/pages/document.yml:58]: I have a few suggestion regarding the tootips here:

* I find that "Open xyz mode" sounds a bit technical, also when you go into one of the "modes", the focus header titel is just simply "Comments" or "Version history", so maybe we could change the tooltips accordingly to "Show comments"/"Show version history"/"Edit document"
* As an anonymous user, I get the "Log in in order to xyz" tooltip text for all disabled icons except for the favorite icon (which still just says "Add to favorites"). For even more consistency we could also change this one to "Log in in in order to add the document to your favorites" or similar?

**Comment by ahelmberger** [comment]
Looks so nice :heart_eyes: I'm really thrilled about the changes on the page and cannot wait to see the whole picture!

**Comment by ahelmberger** [comment]
> When I check out this branch, it re-introduces code from the main branch at tag `v0.110.4` (which were later reverted on `main`). These changes do not appear in the "Files changed" on this PR, though grimacing Could you please rebase or merge the latest `main` into this branch. (You can check if the warning logging in `registered-plugin.js` disappears)

Sorry, I'm stupid. I forgot how git works, please ignore the comment! The old "changes" will not be merged of course.

**Comment by adrianaluchian** [comment]
> > When I check out this branch, it re-introduces code from the main branch at tag `v0.110.4` (which were later reverted on `main`). These changes do not appear in the "Files changed" on this PR, though grimacing Could you please rebase or merge the latest `main` into this branch. (You can check if the warning logging in `registered-plugin.js` disappears)
> 
> Sorry, I'm stupid. I forgot how git works, please ignore the comment! The old "changes" will not be merged of course.

No worries, the branch was pretty behind, it's good to double check it on top of current `main`.
I've rebased it.

File: src/components/pages/document.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/document.yml]: Both good points, I've followed the suggestions for both.
Please double check 🙏

Comment (ahelmberger) [reply src/components/pages/document.yml]: :+1:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**